tests/integration/bootstrap/RestContext.php

```diff
@@ -34,8 +34,8 @@ class RestContext implements Context
     private $headers           = [
         'Accept' => 'application/json'
     ];
-    private $postFields        = array();
-    private $postFiles         = array();
+    private $postFields = array();
```

Comment:
this file changes because of formatting changes, nothing in the behavior is affected